server/utils.js

```diff
@@ -0,0 +1,29 @@
+//Probably should move more stuff into here
+
+var shortid = require("shortid")
+var logger = require("./logger")
+
+var slugify = function(str) {
```

Comment:
Most libraries that require slugs tend to use [String.js's implementation](http://stringjs.com/#methods/slugify) because it handles nice things like accents well.  Up to you if we want to roll our own, though.